docs/reference/docs/delete.asciidoc

```diff
@@ -89,7 +89,7 @@ setting the routing parameter.
 
 Note that deleting a parent document does not automatically delete its
 children. One way of deleting all child documents given a parent's id is
-to use the <<docs-delete-by-query,Delete By Query API>> to perform a
+to use the <<docs-delete-by-query,Delete By Query API>> to perform an
```

Comment:
Looking at the context, I think it should rather be something like "to perform a delete for..."